.github/workflows/deploy-website.yaml

```diff
@@ -0,0 +1,42 @@
+name: Update `error-prone.picnic.tech` website content
+on:
+  push:
+    branches: [$default-branch]
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+concurrency:
+  group: "pages"
+  cancel-in-progress: true
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3.0.2
+      - name: Configure Github Pages
+        uses: actions/configure-pages@v2.1.1
+      - name: Generate documentation
+        run: bash ./generate-docs.sh
+      - name: Build website with Jekyll
+        uses: actions/jekyll-build-pages@v1.0.5
+        with:
+          source: website/
+          destination: ./_site
+      - name: Validating HTML output
+        uses: athackst/htmlproofer-action@main
+        with:
+          check_external_hash: false
```

Comment:
In some websites a hash is used for JS routing, which is done after loading. Disabling this to reduce false-positives.